packages/app/src/messages/presenters/message.presenter.ts

```diff
@@ -63,15 +63,14 @@ export class MessagePresenter {
   private static getMessageType(
     messageType: AcceptablePureMessageType
   ): MessageType {
-    if (
-      messageType === PureMessageType.QUEUED ||
-      messageType === PureMessageType.OUTBOX
-    ) {
+    if (messageType === PureMessageType.OUTBOX) {
```

Comment:
Please add a test for the `presenter`

Reply:
Test in [005e09b](https://github.com/mudita/mudita-center/pull/981/commits/005e09bffe30858f82fce48686253fc8915cc4ca)